docs/bokeh/source/docs/user_guide/interaction/widgets.rst

```diff
@@ -81,9 +81,20 @@ AutocompleteInput
 The AutocompleteInput widget is a general-purpose text input widget that uses
 a list of possible inputs to provide autocomplete while typing.
 
+By default the ``search_strategy`` property is ``"startswith"``, which will
+match against the start of the possible inputs:
+
 .. bokeh-plot:: __REPO__/examples/interaction/widgets/autocompleteinput.py
     :source-position: below
 
+Changing ``search_strategy`` to ``"includes"`` means that matches against any
+substring of the possible inputs will be shown:
+
+.. bokeh-plot:: __REPO__/examples/interaction/widgets/autocompleteinput_includes.py
+    :source-position: below
+
+Try entering "ysk" to see the different results for each strategy.
```

Comment:
I'm not sure if we need to add a second version with a different configuration here. None of the other widgets do, and the intent of the page is really to just get a baseline demonstration of widget creation and see the resulting widget in the docs. I think we would need to split this page up into a page per-widget before attempting to display multiple configs per-widget, since the page is now already quite long. (Happy to discuss that, in a separate issue).

For now, for here I'd say if you want to change the one autocomplete that gets shown to be a version with `includes` configured, that would be 👍